AGENTS.md

```
@@ -102,4 +270,4 @@ See `qtpass-localization` skill for details.
 
 ## Release
 
-See `qtpass-releasing` skill for release process.
+See `qtpass-releasing` skill for release process.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)